static/src/stylesheets/module/facia/item-layouts/_fc-item--three-quarters.scss

```diff
@@ -65,6 +65,7 @@ Three quarter item. Looks like a wide standard, a bit like this:
         .fc-item__content {
             width: auto;
             max-width: gs-span(5);
+            @include flex-basis(initial);
```

Comment:
Ok I'll fix that. The issue is caused by flex, so non-flex should be fine. It's a pattern that we do for others items. e.g [`fc-item__full-50`](https://github.com/guardian/frontend/blame/507b2e2293a00e35353fdd62007f426f3d369722/static/src/stylesheets/module/facia/item-layouts/_fc-item--full-media-50.scss#L67)